run_page/keep_sync.py

```diff
@@ -161,31 +178,34 @@ def parse_raw_data_to_nametuple(
     return namedtuple("x", d.keys())(*d.values())
 
 
-def get_all_keep_tracks(email, password, old_tracks_ids, with_download_gpx=False):
+def get_all_keep_tracks(
+    email, password, old_tracks_ids, keep_sports_data_api, with_download_gpx=False
```

Comment:
I think keep_sports_data_api is not a good name, it confuse me.

Reply:
In fact, it is named this way because the variable is intended to be used to assign a value to RUN_DATA_API. Hence the inevitable ambiguity.